net.kubux.Muehle.yml

```diff
@@ -0,0 +1,23 @@
+app-id: net.kubux.Muehle
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: muehle
+modules:
+  - name: muehle
+    buildsystem: simple
+    build-commands:
+      - tar -xjpvf muehle.tbz -C /
```

Comment:
so you install a binary and build a flatpak for both x86_64 and aarch64. Not sure that will work well.

Reply:
My intent was not to build for x85_64 and aarch64. In fact, I don't want to build for aarch64 as I could not test that anyway. What do I need to say in the manifest to prevent the build?

Reply:
I learned about flathub.json and committed a file that should disable the aarch64 build.

Reply:
you could otherwiser build everything on the flathub infrastructure. aarch64 would just work.

it's disappointing to not build aarch64 for no apparent reason.

Reply:
The reason is that I cannot test the resulting binary. I don't own such hardware. I would not be able to support that built and fix bugs.

Reply:
it sure doesn't open the door to that.

Reply:
Well, the source code is not available online; and at this time, I have no plans of changing that. That is the rationale behind the build command.